packages/ember-principled-forms/package.json

```diff
@@ -51,25 +54,25 @@
     "ember-cli-eslint": "^4.2.3",
     "ember-cli-htmlbars-inline-precompile": "^1.0.3",
     "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-qunit": "^4.1.1",
     "ember-cli-shims": "^1.2.0",
     "ember-cli-sri": "^2.1.0",
-    "ember-cli-typescript": "1.4.0",
+    "ember-cli-typescript-blueprints": "^2.0.0",
     "ember-cli-uglify": "^2.0.0",
     "ember-disable-prototype-extensions": "^1.1.2",
     "ember-export-application-global": "^2.0.0",
     "ember-load-initializers": "^1.0.0",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-object-update": "^0.5.1",
+    "ember-qunit": "^4.4.1",
     "ember-resolver": "^4.0.0",
-    "ember-source": "~3.8.0",
+    "ember-source": "^3.9.1",
     "ember-source-channel-url": "^1.0.1",
     "ember-try": "^0.2.23",
     "eslint-plugin-ember": "^5.0.0",
     "eslint-plugin-node": "^6.0.1",
     "loader.js": "^4.2.3",
     "prettier-eslint": "8.8.2",
-    "typescript": "^3.0.1",
+    "typescript": "^3.4.5",
```

Comment:
Just checking b/c it was an issue for so long – this works correctly and doesn't destroy compile performance when you're using the `Form` and `Field` types?

Reply:
We're not actively using this lib in MWC so we may need to circle back to this if compile perf is a problem 😬

Reply:
I will say that, anecdotally, the build didn't seem to be any slower on Windows than it normally is.

Reply:
@dustinsoftware @tansongyang I confirmed the 3.4.0 nightlies that fixed the underlying perf regression on conditional types we saw throughout the 3.1–3.3 releases and it seemed back to normal, so you should probably be okay; I just wanted to double-check since I saw the upgrade happening!

Reply:
@chriskrycho Always thankful for the help, Chris 😄